Source/ACE.Entity/Position.cs

```diff
@@ -1,38 +1,38 @@
-﻿namespace ACE.Entity
-{
-    using System;
-    using System.IO;
-    using Enum;
-    using Common;
-    using MySql.Data.MySqlClient;
+﻿using System;
+using System.IO;
+using ACE.Entity.Enum;
+using ACE.Common;
+using MySql.Data.MySqlClient;
 
+namespace ACE.Entity
+{
     [DbTable("ace_position")]
     [DbGetList("ace_position", 23, "aceObjectId")]
-    public class Position
+    public sealed class Position
     {
         private LandblockId landblockId;
 
         public LandblockId LandblockId
         {
-            get
-            {
-                return new LandblockId(Cell);
-            }
+            get { return landblockId.Raw != 0 ? landblockId : new LandblockId(Cell); }
             set
             {
                 landblockId = value;
             }
         }
 
         [DbField("aceObjectid", (int)MySqlDbType.UInt32, Update = false, IsCriteria = true)]
-        public virtual uint AceObjectId { get; set; }
+        public uint AceObjectId { get; set; }
```

Comment:
I'd prefer this to be nullable, because (logically speaking) a position can exist without a corresponding AceObject.  A position is just a set of coordinates - this property is something we have to represent an optional relationship.